gtk/src/auto/functions.rs

```diff
@@ -135,11 +135,6 @@ pub fn accelerator_parse(accelerator: &str) -> (u32, gdk::ModifierType) {
     }
 }
 
-//#[doc(alias = "gtk_accelerator_parse_with_keycode")]
-//pub fn accelerator_parse_with_keycode(accelerator: &str, accelerator_codes: Vec<u32>) -> (u32, gdk::ModifierType) {
```

Comment:
you can copy this one from gtk4 if you want :)

Reply:
Done, thanks! :)